infra/modules/s3/pdf-processing-lambda

```diff
@@ -0,0 +1,168 @@
+const AWS = require("aws-sdk");
+const fetch = require("node-fetch");
+
+AWS.config.update({ region: process.env.AWS_REGION });
```

Comment:
is this still needed?

Reply:
This has all been adjusted in the latest file.